fitbenchmarking/fitbenchmarking
# Fixing table tests

#### Description of Work

Fixes #608 

Updates expected html tables to new versions of pandas and other dependences.

#### Testing Instructions

1. Check the tests pass on travis

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
